ShopperInsights/src/main/java/com/braintreepayments/api/ShopperInsightsClient.kt

```diff
@@ -66,18 +68,44 @@ class ShopperInsightsClient @VisibleForTesting internal constructor(
             return
         }
 
-        paymentReadyAPI.processRequest(request)
+        // TODO: get correct merchant ID from SDK
+        val merchantId = "MXSJ4F5BADVNS"
+
+        // Default values
+        val countryCode = "US"
```

Comment:
Similar Q for the countryCode - I thought we could omit this one based on one of our Slack threads with Harish (and that only currencyCode was required)

Reply:
When testing using postman I needed country code to get Venmo recommendation back.

Reply:
Thanks for confirming!